src/components/css/RowsPerPageToggleButton.css

```diff
@@ -0,0 +1,10 @@
+.toggle-button {
+    border-radius: unset !important;
```

Comment:
is it a good idea to do !important?

Reply:
the default toggle-button here has a round border, has to use important to change that. 
Maybe it's not a good idea... but I don't know if there is another way to get around this.

Reply:
I solve this.